docs/api/rpc.md

```diff
@@ -1,374 +1,592 @@
 ---
 id: rpc
-title: API
-sidebar_label: API
+title: RPC
+sidebar_label: RPC
 ---
 
-Standard JSON RPC 2.0 is used across the board.
+## Setup
 
-Notes:
+- `POST` for all methods
+- `JSON RPC 2.0`
+- `id: "dontcare"`
+- endpoint URL varies by network:
+  - MainNet `https://rpc.mainnet.near.org`
+  - TestNet `https://rpc.testnet.near.org`
+  - BetaNet `https://rpc.betanet.near.org` _(may be unstable)_
 
-- all methods are HTTP `POST`
-- endpoint URL varies by network
-  - for MainNet use `https://rpc.mainnet.near.org`
-  - for TestNet use `https://rpc.testnet.near.org`
-  - for BetaNet use `https://rpc.betanet.near.org` _(may be unstable)_
+You can see this interface defined in `nearcore` [here](https://github.com/nearprotocol/nearcore/blob/master/chain/jsonrpc/client/src/lib.rs#L185).
 
-You can see this interface defined in `nearcore` [here](https://github.com/nearprotocol/nearcore/blob/master/chain/jsonrpc/client/src/lib.rs#L185)
+An easy way to test the examples below, would be to use an API request tool such as [Postman](https://www.postman.com/). You will only need to configure two things:
 
-<blockquote class="info">
-<strong>note</strong><br><br>
+  1) Make sure you add a header with a key of `Content-Type` and value of `application/json`.
+  ![postman-setup-header](/docs/assets/postman-setup-headers.png)
 
-The commands mentioned on this page beginning with `http` are using the tool [HTTPie](https://httpie.org/doc#installation).
-Please install it in order to copy/paste those commands.
+  2) Then select the `Body` tab and choose the `raw` radio button and ensure `JSON` is the selected format.
+   ![postman-setup-header](/docs/assets/postman-setup-body.png)
 
-</blockquote>
+After that is setup, just copy/paste the `JSON object` example snippets below into the `body` of your request, on Postman, and click `send`.
 
-## Status
-
-`status`
-
-Returns current status of the node
-
-- method `query(path: string, data: string)`
-- params *none*
+If you prefer to use a command line interface, such as [HTTPie](https://httpie.org/) the format is as follows:
 
 ```bash
-http post https://rpc.testnet.near.org jsonrpc=2.0 method=status params:='[]' id=dontcare
+http post https://rpc.testnet.near.org jsonrpc=2.0 id=dontcare method=query params:='{}' 
 ```
+Note: For HTTPie, params use either an object or array as a string.
 
+## Accounts / Contracts
+
+### View account
+ Returns view of account information.
+  - method: `query`
+  - params 
+     - `request_type`: `view_account`
+     - `finality`: `optimistic`, `near-final`, or `final`
```

Comment:
Let's avoid mentioning `near-final` in our documentation. It may create unnecessary confusion